packages/lisk-transactions/test/utils/validation/validate_transaction.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import fixtures from '../../../fixtures/transactions.json';
+import invalidFixtures from '../../../fixtures/invalid_transactions.json';
+import validateTransaction from '../../../src/utils/validation/validate_transaction';
+
+describe('validateTransaction', () => {
+	describe('#validateTransaction', () => {
+		describe('when fixtures nad invalid fixtures provided', () => {
```

Comment:
How about two suites: `when valid fixtures are provided` and `when invalid fixtures are provided`?